### DISCUSSION TITLE
Apple Silicon Docker Images for new Macs

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#5441] [General]
Is it possible / feasible to have apple silicon docker images for budibase ?

**Comment by shogunpurple** [comment]
Hi @poirazis - were you having some problems running the platform on your silicon mac?

I use an apple silicon mac myself, and I'm able to run the platform by using the following workaround to build the couchDB image.

<img width="839" alt="Screenshot 2022-04-19 at 15 00 25" src="https://user-images.githubusercontent.com/11256663/164021495-85c646fa-5e56-40e7-a468-f20caeb65c28.png">

Would this work for you?

**Reply by poirazis** [reply]
I didn't make this change when i initially installed Budibase.
Can i make this change now or do i need to install again ?

The platform is working, but i get the following in Docker.
<img width="532" alt="Screenshot 2022-04-19 at 5 26 26 PM" src="https://user-images.githubusercontent.com/80512170/164030641-22b0dd3e-c176-4409-bd6e-d28d8c125f54.png">
